dev/io.openliberty.jakarta.xmlBinding.4.0/bnd.bnd

```diff
@@ -31,12 +39,17 @@ DynamicImport-Package: \
   org.glassfish.jaxb.*
 
 Include-Resource:\
-  @${repo;jakarta.xml.bind:jakarta.xml.bind-api;4.0.0;EXACT}!/!(META-INF/maven/*|module-info.class)
+  @${repo;jakarta.xml.bind:jakarta.xml.bind-api;4.0.0;EXACT}!/!(META-INF/maven/*|module-info.class), \
+  io/openliberty/jakarta=${bin}/io/openliberty/jakarta
+  jakarta/xml/bind=${bin}/jakarta/xml/bind
+
+Multi-Release: true
```

Comment:
It is not multi release which is why it was not there.  It shouldn't be there unless you are adding multi release function for Java 11 and 17 somehow.